src/www/system_advanced_firewall.php

```diff
@@ -738,13 +738,13 @@
                   <input name="noantilockout" type="checkbox" value="yes" <?= empty($pconfig['noantilockout']) ? '' : 'checked="checked"' ?>/>
                   <?= gettext('Disable administration anti-lockout rule') ?>
                   <div class="hidden" data-for="help_for_noantilockout">
-                    <?= sprintf(gettext("When this is unchecked, access to the web GUI or SSH " .
-                                "on the %s interface is always permitted, regardless of the user-defined firewall " .
-                                "rule set. Check this box to disable the automatically added rule, so access " .
-                                "is controlled only by the user-defined firewall rules. Ensure you have a firewall rule " .
-                                "in place that allows you in, or you will lock yourself out."),
-                                count($config['interfaces']) == 1 && !empty($config['interfaces']['wan']['if']) ?
-                                gettext('WAN') : gettext('LAN')) ?>
+                    <?= gettext(<<<EOD
```

Comment:
this breaks translations. not sure why that's a good idea

Reply:
reverted